resource-usage-collect-addon/helm/templates/placement.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: cluster.open-cluster-management.io/v1beta1
+kind: Placement
+metadata:
+  name: {{ .Values.placement.name }}
+  namespace: {{ .Values.placement.namespace }}
+spec:
+  clusterSets:
+    - global
+
```

Comment:
Also need file https://github.com/open-cluster-management-io/addon-contrib/blob/main/resource-usage-collect-addon/deploy/resources/managed-cluster-set-binding.yaml to create a ManagedClusterSetBinding, otherwise the Placement won't select any clusters.